controllers/core/openstackcontrolplane_controller.go

```diff
@@ -249,6 +256,15 @@ func (r *OpenStackControlPlaneReconciler) reconcileNormal(ctx context.Context, i
 		return ctrlResult, nil
 	}
 
+	if instance.Spec.Ironic.Enabled {
```

Comment:
Apologies for doing this to you. But the general consensus on the implementation for the other change was the keep the current design standard. So the preference from the wider team is that we just call reconcile here and leave the `if instance.Spec.Ironic.Enabled {` check within in the `ReconcileIronic()` function. Which is what you had originally done.

We should probably change that back before we merge this to satisfy the concerns that were raised on:
https://github.com/openstack-k8s-operators/openstack-operator/pull/162

Reply:
Done